src/ValidationPackages/ValidationPackage.fs

```diff
@@ -22,6 +22,15 @@ type ValidationPackage(name, ?version) =
     member this.Copy() =
         ValidationPackage.make this.Name this.Version
 
+    /// Pretty printer 
+    override this.ToString() =
```

Comment:
Maybe do not use yaml style for pretty printing, or use YAMLicious to generate this. But i would prefer typical f# printing syntax

Reply:
Also, i think the to string is valuable here because it represents something that only ever exists as a yaml file in the arc.

Reply:
It is just that the ToString function should not be responsible for YAML formatting, this should be a different function.

Reply:
it is not. it is just for printing. formatting for actual yaml export is done with yamlicious.

Reply:
Then why do we have tests for this? 😮